samples/ios/app/glean-sample-app.xcodeproj/xcshareddata/xcschemes/glean-sample-app.xcscheme

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
```

Comment:
Mh, why this new file? Why was it missing before?

Reply:
It apparently was..  I'm not sure where the previous one when but it's just a Xcode file that tells Xcodebuild which build configurations to use.